android/src/com/frostwire/android/core/Constants.java

```diff
@@ -52,7 +52,13 @@ private Constants() {
     public static final String PREF_KEY_CORE_LAST_SEEN_VERSION = "frostwire.prefs.core.last_seen_version";
 
     public static final String PREF_KEY_NETWORK_ENABLE_DHT = "froswire.prefs.network.enable_dht";
+    /**
+     * @deprecated left for migration purposes, use {@link #PREF_KEY_NETWORK_USE_WIFI_ONLY }
+     */
+    @Deprecated
     public static final String PREF_KEY_NETWORK_USE_MOBILE_DATA = "frostwire.prefs.network.use_mobile_data";
+    public static final String PREF_MIGRATION_FINISHED_KEY_NETWORK_USE_MOBILE_DATA_TO_USE_WIFI_ONLY = "frostwire.prefs.migration.use_mobile_data.use_wifi_only";
```

Comment:
Well, we need the migration to run once. So somehow we need to mark that it's been completed use the new preference